pkg/cmd/stop/stop_pipeline.go

```diff
@@ -221,6 +222,13 @@ func (o *StopPipelineOptions) cancelPipelineRun() error {
 		if pr == nil {
 			return fmt.Errorf("no PipelineRun found for name %s", a)
 		}
+		pr, err = pipelines.Get(pr.Name, metav1.GetOptions{})
+		if err != nil {
+			return errors.Wrapf(err, "getting PipelineRun %s", pr.Name)
+		}
+		if tekton.PipelineRunIsComplete(pr) {
+			return fmt.Errorf("the PipelineRun %s has already completed", a)
```

Comment:
Is this really an error condition?

Reply:
That test was performed earlier when listing the possible pipelines to stop. But since the status may have evolved since, the user may have selected a pipeline that already completed. I chose it to be an error, because if you wanna cancel something, you'd be surprised is completed already. I'm not clear if it should be (could also be logged), but this is a rather unlikely case anyway, that will happen much less often than the current error.

Reply:
> I chose it to be an error, because if you wanna cancel something, you'd be surprised is completed already. I'm not clear if it should be (could also be logged)

I would be inclined to just log. My point of view is that as user your goal is to stop a pipeline. 
As long as the outcome of my command is that the pipeline ended, I'd considered it successful. An error for me would be if the pipeline would still be running.

Returning an error does not only have implications on what's displayed in the end on the screen, but also the return status of the command. Returning an error also means that the return code is non 0 which seems wrong if I think about using `stop pipeline` in a script.

Reply:
Sounds reasonable. Change pushed.

Reply:
thanks